_opensearch/segment-replication/configuration

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Segment Replication Configuration
+nav_order: 122
+parent: Segment Replication Feature
+grand_parent: Opensearch
+---
+
+## Segment Replication Configuration
```

Comment:
Per our style guide, we need a sentence with a simple description that separates the headings.

Reply:
kk